astroquery/atomic/core.py

```diff
@@ -39,7 +41,9 @@ def query_object(self, wavelength_range=None, wavelength_type=None,
                      show_auto_ionizing_transitions=None,
                      output_columns=('spec', 'type', 'conf',
                                      'term', 'angm', 'prob',
-                                     'ener')):
+                                     'ener'),
+                     cache=True,
```

Comment:
Totally optional: maybe this is a good time to make kwargs kwarg only? e.g right from the beginning?

Reply:
I'd rather punt this to a different PR